src/main/kotlin/apply/ui/admin/selections/EvaluationAnswerForm.kt

```diff
@@ -0,0 +1,35 @@
+package apply.ui.admin.selections
+
+import apply.application.EvaluationAnswerData
+import com.vaadin.flow.component.select.Select
+import com.vaadin.flow.component.textfield.TextField
+import dev.mett.vaadin.tooltip.Tooltips
+import support.views.BindingIdentityFormLayout
+
+class EvaluationAnswerForm() : BindingIdentityFormLayout<EvaluationAnswerData>(EvaluationAnswerData::class) {
+    val score: Select<Int> = Select(0)
+    private val title: TextField = TextField().apply { isReadOnly = true }
+
+    init {
+        add(title, score)
+        drawRequired()
+    }
+
+    constructor(title: String, description: String, maximumScore: Int) : this() {
+        this.score.setItems((0..maximumScore).toList())
+        this.title.value = title
+        Tooltips.getCurrent().setTooltip(this.title, description)
```

Comment:
```suggestion
        val config = TooltipConfiguration(description)
        config.interactive = true
        Tooltips.getCurrent().setTooltip(this.title, config)
```
툴팁 설정중에 이런 값을 주니 툴팁 내부에 드래그도 할수 있더라고요
코드는 좀 못생겨지네요

Reply:
바로 반영했습니다! 원하는대로 동작하네요 👍